decidim-home/app/controllers/decidim/pages_controller.rb

```diff
@@ -16,7 +16,8 @@ class PagesController < ApplicationController
         authorize_resource :public_pages, class: false
         delegate :page, to: :page_finder
         helper_method :page, :next_meetings, :promoted_participatory_processes, :highlighted_participatory_processes, :stats, :home_participatory_processes, :searchProcesses, :searchFeatures, :searchMeetings, :searchRegulations, :searchAssemblies
-
+        helper Decidim::SanitizeHelper
```

Comment:
If `PagesController` inherits from `ApplicationController` isn't this line redundant? Better remove it.

Reply:
Now I see that I saw the same helper in decidim-process-extended/app/controllers/decidim/process/extended/application_controller.rb

is `PagesController` inheriting from this `ApplicationController` or we have more than 1 of those?

Reply:
Existeix més d'un ApplicationController, i en aquest cas, si no vaig errat, està agafant el de la pròpia aplicació. No el del Decidim.